python/cudf/setup.py

```diff
@@ -30,16 +30,16 @@
 import versioneer
 
 install_requires = [
```

Comment:
This list was getting unwieldy so I sorted it. The only changes are removing Cython and adding cuda-python.